src/Commands/Configure.ts

```diff
@@ -1,34 +1,24 @@
-import { logger } from '../Utils';
+import { Helpers, logger } from '../Utils';
 import { ConnectivityUrls, Platform, TestType } from '../Wizard';
 import { container } from '../Config';
 import { Arguments, Argv, CommandModule } from 'yargs';
 import { URL } from 'url';
 
 export class Configure implements CommandModule {
```

Comment:
Don't work as expected, there's no way to use `--tcp`, `--http` and `--auth` options

Reply:
not relevant